docs/releases/1.9.txt

```diff
@@ -794,6 +794,15 @@ no longer include the scheme and domain part of the URLs. For example,
 replaced by ``self.assertRedirects(response, '/some-url/')`` (unless the
 redirection specifically contained an absolute URL, of course).
 
+If you still need the old behavior, you can restore it by writing a :doc:`custom
```

Comment:
My use case is an ancient Apache version with mod_scgi that interprets relative redirects as "internal redirect". Do you think that is worth mentioning?